backend/src/main/java/planit/people/preparation/DAOs/IDAO_Google_account.java

```diff
@@ -1,7 +1,18 @@
 package planit.people.preparation.DAOs;
 
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 import planit.people.preparation.Entities.Entity_Google_account;
+import planit.people.preparation.Entities.Entity_User;
+
+import java.util.List;
+import java.util.Set;
 
 public interface IDAO_Google_account extends CrudRepository<Entity_Google_account, Long> {
+    @Query("SELECT new Entity_Google_account(google.id, google.email, google.the_user, google.refresh_token) FROM Entity_Google_account google  WHERE google.email IN  (:emails)")
+    List<Entity_Google_account> getEntityGoogleAccountByEmail(@Param("emails") List<String> emails);
```

Comment:
What does this method do? It's called "Get Entity [...] by email" but returns a list and takes a list

Reply:
the param is a list of attendee emails and returns to us the list of entity google account from which we get planIt user id then we use the second function getByPlanItUserId